sdks/python/apache_beam/transforms/core.py

```diff
@@ -1907,7 +1907,9 @@ def FlatMap(fn = identity, *args, **kwargs):  # pylint: disable=invalid-name
 
   The callable must return an iterable for each element of the input
   :class:`~apache_beam.pvalue.PCollection`. The elements of these iterables will
-  be flattened into the output :class:`~apache_beam.pvalue.PCollection`.
+  be flattened into the output :class:`~apache_beam.pvalue.PCollection`. If
+  no callable is given, then all elements of the input PCollection must already
+  be iterables themselves and will be flattened into the output PCollection.
```

Comment:
Thank you! could you please also update https://beam.apache.org/documentation/transforms/python/elementwise/flatmap/ ? If you are interested, feel free to add an example of flattening lists nested in a pcoll (no need to strictly follow the 'gardening' theme).

Reply:
Inserted a new example pretty early on the page since it's likely more relevant for most users than the later ones. Happy to adjust or reorder though

Reply:
good call, thanks!